pkgs/tools/X11/devour/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, libX11 }:
+stdenv.mkDerivation rec {
+  pname = "devour";
+  version = "12";
+
+  src = fetchFromGitHub {
+    owner = "salman-abedin";
+    repo = "devour";
+    rev = version;
+    sha256 = "1qq5l6d0fn8azg7sj7a4m2jsmhlpswl5793clcxs1p34vy4wb2lp";
+  };
+
+  installPhase = ''
+    install -Dm555 -t $out/bin devour
+  '';
+
+  buildInputs = [ libX11 ];
+
+  meta = with stdenv.lib; {
+    description = "Devour hides your current window before launching an external program and unhides it after quitting.";
```

`meta.description` should not end with a period, and this seems a bit long.
```suggestion
    
    description = "Devour hides your current window when launching an external program";
    longDescription = "Devour hides your current window before launching an external program and unhides it after quitting";